webpack.config.js

```diff
@@ -50,6 +50,7 @@ module.exports = (env) => {
       bundlesLandingPage: 'pages/bundles-landing/bundlesLanding.jsx',
       monthlyContributionsPage: 'pages/monthly-contributions/monthlyContributions.jsx',
       monthlyContributionsThankyouPage: 'pages/monthly-contributions-thankyou/monthlyContributionsThankyou.jsx',
+      monthlyContributionsExistingPage: 'pages/monthly-contributions-existing/monthlyContributionsExisting.jsx',
```

Comment:
After an offline discussion, we decide to do research about commons chunks and refactor in the near future. Answering the question above, yes we would need to import two scripts, the shared script and the non-shareable one.